build/tools/amebasmart/gnu_utility/rtk_data_binary.py

```diff
@@ -32,8 +35,18 @@ def text_to_binary(input_file, output_file):
             # Write binary data to the output file
             outfile.write(binary_data)
 
-if __name__ == "__main__":
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../../os/tools')))
+import config_util as util
+CONFIG_SECOND_FLASH_PARTITION = util.get_value_from_file(cfg_file, "CONFIG_SECOND_FLASH_PARTITION=").rstrip('\n')
+if CONFIG_SECOND_FLASH_PARTITION == "y":
     input_file = dirpath + "/rtk_data_binary.txt"
     output_file = dirpath + "/rtk_ext_flash_data.bin"
 
     text_to_binary(input_file, output_file)
+    # Ext flash valid
+    print("1")
```

Comment:
Could you change the log to correct message?

Reply:
This print is to send the value back to the bash. You can find sys_exit(0) at below. It will be used to determine whether we pass the binary to do prepend header.